<launch>


        
    <!-- <node name="republishL" type="republish" pkg="image_transport" output="screen" args="raw in:=/camera/color/image_rect_color compressed out:=/camera/color/image_rect_color" /> -->

	<node name="coppelia_combiner" pkg="visavis_vision" type="coppelia_combiner" args="" respawn="false"
		output="screen">

		<!-- if false, it work on coppeliasim with rosInterface plugin-->
		<param name="old_plugin" type="bool" value="false" />

		<!-- subscribers topic-->
		<param name="camera_color_name" type="string" value="/camera/color/image_rect_color" />
		<param name="camera_depth_name" type="string"
			value="/camera/aligned_depth_to_color/image_raw" />
		<param name="depth_buffer_in" type="string" value="/coppelia/depth_buff" />
		<param name="point_cloud_out" type="string" value="/camera/depth_registered/points" />


		<param name="u_res" value="480" />
		<param name="v_res" value="640" />
		<param name="near_clip" value="1.00e-02" />
		<param name="far_clip" value="5.00e+00" />
		<param name="view_angle" value="60.0" />
	</node>


    <node name="colorSeg" pkg="util_pkg" type="color_segmentation" args="" respawn="false" output="screen">
                
        <param name="cld_topic_name" type="string" value="/camera/depth_registered/points"/>
        <param name="fixed_frame" type="string" value="world"/>
        <param name="optical_frame" type="string" value="camera_color_optical_frame"/>
        <param name="voxel_size" value="0.001"/>
       

        <rosparam param="hsv_range"> [200,280,0.20,1] </rosparam> 
    </node>


</launch>	
    